runtime/tests/evm_precompiles_erc721.rs

```diff
@@ -177,7 +176,7 @@ fn erc721_approve_and_transfer() {
 		// Approval should be removed
 		assert_eq!(
 			TokenApprovals::erc721_approvals((collection_id, series_id, serial_number)),
-			AccountId::default()
+			H160::from_slice(&hex!("0000000000000000000000000000000000000000"))
```

Comment:
this is clearer for these lines
```suggestion
			H160::default()
```